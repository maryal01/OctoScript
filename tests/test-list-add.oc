list<string> l = ["manish", "danielle"];
list<string> new_list = l.list_add("conor");
int i = 0;
while(i < new_list.list_length()){
    print(new_list.list_get(i));
    print("\n");
    i = i + 1;
}

new_list = l.list_add("sinan");
i = 0;
while(i < new_list.list_length()){
    print(new_list.list_get(i));
    print("\n");
    i = i + 1;
}

new_list = new_list.list_add("conor");
i = 0;
while(i < new_list.list_length()){
    print(new_list.list_get(i));
    print("\n");
    i = i + 1;
}
