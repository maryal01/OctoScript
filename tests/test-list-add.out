manish
danielle
conor
manish
danielle
sinan
manish
danielle
sinan
conor
